### PR TITLE
rename currency command to `currency add`

### PR DESCRIPTION
**Pull request by maxtaco**
- also add a deprecated BTC command

### CODE REVIEW

**Comment by maxtaco** [comment]
@malgorithms as we discussed.

**Comment by maxtaco** [comment]
@patrickxb does this look ok?  If so, I have to fix `kbfs-server`

File: go/client/cmd_currency.go [hunk 1]
```diff
@@ -4,25 +4,66 @@
...
 	address string
 	force   bool
 	wanted  string
 }
 
-func (c *CmdCurrency) ParseArgv(ctx *cli.Context) error {
+func NewCmdCurrency(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "currency",
+		Usage:        "Add/delete cryptocurrency addresseses",
```
Comment (patrickxb) [inline go/client/cmd_currency.go]: just Add until we have delete?

**Comment by maxtaco** [comment]
ok fixed

**Pull request merged**